client/components/systems/fate/consequences.vue

```diff
@@ -138,12 +140,6 @@
         methods: {
             openEdit()
             {
-                // // Reset the edit fields
-                // this.mildConsequence1Edit = _.cloneDeep(this.mildConsequence1);
-                // this.mildConsequence2Edit = _.cloneDeep(this.mildConsequence2);
-                // this.moderateConsequenceEdit = _.cloneDeep(this.moderateConsequence);
-                // this.severeConsequenceEdit = _.cloneDeep(this.severeConsequence);
-
```

Comment:
Commented out code.

Reply:
That was me removing commented out code. ;)

Reply:
Oh god you’re right! Apparently I’m color blind now.